src/upgrade/upgrade_adapter.ts

```diff
@@ -22,7 +22,7 @@ export class NgMetadataUpgradeAdapter {
      *
      * E.g. `upgradeAdapter.bootstrap(AppComponent, providers)`
      */
-    this.bootstrap = createBootstrapFn(this._upgradeAdapter.bootstrap.bind(this._upgradeAdapter));
+    // this.bootstrap = createBootstrapFn(this._upgradeAdapter.bootstrap.bind(this._upgradeAdapter));
```

Comment:
This is resolved in the ngUpgrade PR (which was branched from this branch), so it will not be an issue by the time this PR is merged into 3.x. It was purely an interim thing to avoid tsc issues I think